ChangeLog.md

```diff
@@ -0,0 +1,42 @@
+## Changes in 1.0.1
+
+* [gui.h](include/clap/ext/gui.h): fix doc: set_scale must be provided
+* [events.h](include/clap/events.h): remove `clap_event_type` which was never used
+* [draft/cv.h](include/clap/ext/draft/cv.h): rename `CLAP_EXT_CV` to `CLAP_EXT_TRANSPORT_CONTROL`
+* [draft/tuning.h](include/clap/ext/draft/tuning.h): rename `clap_client_tuning` to `clap_plugin_tuning`
+
+## Changes in 1.0.0
+
+# new stable interfaces
```

Comment:
I'd capitalize these two sections since the other changelog sections do the same thing, but that's just a style nitpick. Otherwise, looks good to me!

Reply:
Capitalize just the first word or each?

Reply:
Yes, just the 'New'.